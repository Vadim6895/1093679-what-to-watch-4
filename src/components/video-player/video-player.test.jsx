import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const filmCards = [{
  movieName: `The Grand Budapest Hotel`,
  productionDate: `1984`,
  genre: `Drama`,
  moviePoster: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  src: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  id: 0,
  director: `Anthony Mann`,
  actors: `Anthony Mann`,
  rating: 9,
  ratingsQuantity: 250,
  description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum .`,
  length: 120,
  reviews: [{
    text: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum dolor sit amet, consectetur adipiscing elit.Lorem ipsum .`,
    rating: 8,
    name: `Anthony Mann`,
    date: new Date(),
  }]
}];

it(`Should VideoPlayer render correctly`, () => {
  const tree = renderer
  .create(<VideoPlayer
    src={filmCards[0].src}
    moviePoster={filmCards[0].moviePoster}
    onMouseEnter={() => {}}
    onMouseLeave={() => {}}
    isPlaying={false}
  />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
