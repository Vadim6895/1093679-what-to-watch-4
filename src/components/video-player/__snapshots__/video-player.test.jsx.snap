// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should VideoPlayer render correctly 1`] = `
<video
  className="small-movie-player"
  height="175"
  muted={true}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
  width="280"
/>
`;
